core/src/main/resources/hudson/lifecycle/WindowsInstallerLink/index.jelly

```diff
@@ -26,6 +26,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
   <l:layout permission="${app.ADMINISTER}" title="${%Install as Windows Service}">
     <st:include it="${app}" page="sidepanel.jelly" />
+    <l:breadcrumb title="${%Install as Windows Service}" />
```

Comment:
❌ This one should be removed.

We are getting two `li` instead of one. There is no index page for that one.

<details><summary>Screenshot</summary>

![Screenshot_2022-11-29_194338_001](https://user-images.githubusercontent.com/2662497/204618435-e473d3f7-39a6-4284-8377-6d402290504e.png)


</details>